akka-stream/src/main/scala/akka/stream/javadsl/SubFlow.scala

```diff
@@ -961,7 +961,7 @@ class SubFlow[-In, +Out, +Mat](delegate: scaladsl.SubFlow[Out, Mat, scaladsl.Flo
     new SubFlow(delegate.batchWeighted(max, costFn.apply, seed.apply)(aggregate.apply))
 
   /**
-   * Allows a faster downstream to progress independently of a slower publisher by extrapolating elements from an older
+   * Allows a faster downstream to progress independently of a slower upstream by extrapolating elements from an older
```

Comment:
good rename

Reply:
Thanks. I'm thinking about a separate PR that unifies this type of terminology for the entire API.